haas/rest.py

```diff
@@ -164,10 +172,19 @@ def foo(bar, baz, quux):
           whose second is an integer (the status code).
     """
     def register(f):
+
+        if isinstance(methods, list):
```

Comment:
Is there any reason to keep the 'single method' option?  Why not a list with one element---checking if something is a list is not terribly "pythonic" or whatever